### DISCUSSION TITLE
is the memory usage normal?

### DISCUSSION THREAD

**Discussion by rizqicy** [hyprwm/Hyprland#8287] [Q&A] [answered]
Hi All,

anybody experienced this kind of memory usage on your hyprland?
![2024-10-29T14:55:24,274816750+07:00](https://github.com/user-attachments/assets/9028e744-25e3-4028-a18d-932072cc78af)

usage: nvim, dbeaver, firefox+google meeting (10+ tabs)

any advice? or thoughts?

hyprland ver: hyprland-git (d679d200)

**Comment by vaxerski** [comment, marked as answer]
does not look normal

**Reply by rizqicy** [reply]
okay then..need to reproduce it properly bcause right now it appears suddenly and I dont know what is the trigger

**Comment by rizqicy** [comment]
turns out it is due to dbeaver (gtk3 memory bug)
related issue: https://github.com/hyprwm/Hyprland/issues/7038
